src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,27 @@
+import cn from 'classnames';
+import React from 'react';
+
+type Props = {
+  errorMessage: string;
+  onClose: (message: string) => void;
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  errorMessage,
+  onClose,
+}) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={`notification is-danger is-light has-text-weight-normal ${!errorMessage && 'hidden'}`}
```

Comment:
better to use classnames for conditional classes

Reply:
Not fixed

Meant `classnames` library (or any similar):
![image](https://github.com/user-attachments/assets/afd6c241-7b76-4def-be3e-cc287b00620c)